modules/server/src/test/scala/scaladex/server/route/ControllerBaseSuite.scala

```diff
@@ -22,7 +29,26 @@ trait ControllerBaseSuite extends AsyncFunSpec with Matchers with ScalatestRoute
     realConfig.copy(filesystem = realConfig.filesystem.copy(index = index))
   }
 
-  val githubUserSession = new GithubUserSession(config.session)
+  private implicit val cs: ContextShift[IO] =
+    IO.contextShift(ExecutionContext.global)
+
+  private val databaseConfig: PostgreSQLConfig = PostgreSQLConfig
+    .load()
+    .get
+    .asInstanceOf[PostgreSQLConfig]
+
+  val transactor: Transactor.Aux[IO, Unit] =
+    Transactor
+      .fromDriverManager[IO](
+        databaseConfig.driver,
+        databaseConfig.url,
+        databaseConfig.user,
+        databaseConfig.pass.decode
+      )
+
+  val sqlDatabase = new SqlDatabase(databaseConfig, transactor)
+
+  val githubUserSession = new GithubUserSession(config.session, sqlDatabase)
   val githubAuth = MockGithubAuth
 
   val database: InMemoryDatabase = new InMemoryDatabase()
```

Comment:
```suggestion
  val githubAuth = MockGithubAuth
  val database: InMemoryDatabase = new InMemoryDatabase()
   val githubUserSession = new GithubUserSession(config.session, database)
```

Reply:
Actually this is enough to make the tests pass, there is nothing to change in `InMemoruDatabase` or `MockGithubAuth`.
I am pushing the change.